Automattic/jetpack
# Social Previews | Add social icons to the top nav

Completes 1203820933396971-as-1204281808721003/f

## Proposed changes:

* Update social icons from `social-link` block from `@wordpress/block-library`
* Add new services to Social Previews sidebar and modal

### Other information:


## Jetpack product discussion
PT: pdrWKz-Ka-p2

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:






* Checkout this branch
* Start writing a new post.
* Click Jetpack icon on the sidebar
* Under "Social Previews", click on "Preview"
* Confirm that you see the new/updated icons
* Confirm that the navigation works as expected
* Confirm that the design matches Figma - NIq2WbeH5khnwLVaHbUvSX-fi-3113-6000

<table>
	<thead>
		<tr>
			<th>BEFORE</th>
			<th>AFTER</th>
		</tr>
	</thead>
	<tbody>
		<tr>
			<th colspan="2">Desktop</th>
		</tr>
		<tr>
			<td><img width="572" alt="Screenshot 2023-04-03 at 4 28 51 PM" src="https://user-images.githubusercontent.com/18226415/229491289-9aefe1d7-7f00-4ba0-b06e-4dded339e681.png"></td>
			<td><img width="593" alt="Screenshot 2023-04-03 at 4 15 40 PM" src="https://user-images.githubusercontent.com/18226415/229491379-78c2a347-254e-4705-be2b-d6937665d74f.png"></td>
		</tr>
		<tr>
			<td><img width="1388" alt="Screenshot 2023-04-03 at 4 29 07 PM" src="https://user-images.githubusercontent.com/18226415/229491504-7d6e9bb9-75fb-48f8-91cd-b25ccce6118b.png"></td>
			<td><img width="1370" alt="Screenshot 2023-04-03 at 4 16 09 PM" src="https://user-images.githubusercontent.com/18226415/229491568-54183a17-25b4-4580-b53c-1adf04e21070.png"></td>
		</tr>
		<tr>
			<th colspan="2">Mobile</th>
		</tr>
		<tr>
			<td><img width="401" alt="Screenshot 2023-04-03 at 4 30 00 PM" src="https://user-images.githubusercontent.com/18226415/229491717-6c807d8e-5d4f-40ac-8715-29f201b5f46a.png"></td>
			<td><img width="398" alt="Screenshot 2023-04-03 at 4 30 14 PM" src="https://user-images.githubusercontent.com/18226415/229491768-7c6b174c-21ad-4f7f-bb9f-1746d3dcb974.png"></td>
		</tr>
		<tr>
			<td><img width="397" alt="Screenshot 2023-04-03 at 4 31 04 PM" src="https://user-images.githubusercontent.com/18226415/229491820-4ddf7e1d-f654-4480-948c-8d597fe000bb.png"></td>
			<td><img width="400" alt="Screenshot 2023-04-03 at 4 31 15 PM" src="https://user-images.githubusercontent.com/18226415/229491868-bb274c2e-febe-4724-b43c-57776e04d198.png"></td>
		</tr>
	</tbody>
</table>

Keyboard navigation

https://user-images.githubusercontent.com/18226415/229492061-07e196b6-f5fa-467d-970f-00e5da260a43.mov


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1204281808721003
